packages/framework/src/Command/Exception/MissingLocaleException.php

```diff
@@ -25,9 +25,6 @@ public function __construct($missingLocale, Throwable $previous = null)
         parent::__construct($message, 0, $previous);
     }
 
-    /**
-     * @return string
```

Comment:
This should not have been removed!

Reply:
Please report these issues at Symplify so they can be fixed